src/types.ts

```diff
@@ -111,6 +111,12 @@ export declare interface ISidebarInitData extends ICommonInitData {
     type: LocationType.WIDGET;
 }
 
+export declare interface IRTELocationInitData extends ICommonInitData {
+    content_type: ContentType;
+    entry: Entry;
+    locale: string;
+}
+
```

Comment:
We already have `IRTEInitData` type definition for RTE Plugins, we could add entry to the same and make use of that instead.

Reply:
@Nilesh-y can you check this?

Reply:
@Nilesh-y Could you please check this comment ?

Reply:
Yes @abhishek305 @KANE-99 made the changes.... please check

Reply:
got it @Nilesh-y